examples/website/geospatial/app.js

```diff
@@ -80,22 +80,48 @@ export default class App extends PureComponent {
 
   _renderLayer() {
     const {selectedExample, selectedLoader, uploadedFile} = this.state;
+
+    let layerData;
+    if (uploadedFile) {
+      layerData = uploadedFile;
+    } else if (EXAMPLES[selectedLoader][selectedExample]) {
+      layerData = EXAMPLES[selectedLoader][selectedExample].data;
+    } else {
+      layerData = EXAMPLES[INITIAL_LOADER_NAME][INITIAL_EXAMPLE_NAME].data;
+    }
+
     return [
       new GeoJsonLayer({
         id: `geojson-${selectedExample}(${selectedLoader})`,
-        data: uploadedFile
-          ? uploadedFile
-          : EXAMPLES[selectedLoader][selectedExample]
-          ? EXAMPLES[selectedLoader][selectedExample].data
-          : EXAMPLES[INITIAL_LOADER_NAME][INITIAL_EXAMPLE_NAME].data,
+        data: layerData,
         opacity: 0.8,
         stroked: false,
         filled: true,
         extruded: true,
         wireframe: true,
         getElevation: (f) => Math.sqrt(f.properties.valuePerSqm) * 10,
-        getLineColor: [255, 255, 255],
-        pickable: true
+        getLineColor: [255, 0, 0],
+        getLineWidth: 3,
+        lineWidthUnits: 'pixels',
+        pickable: true,
+        // TODO: Why aren't these loadOptions aren't passed for an uploaded file???
+        loadOptions: {
+          geopackage: {
+            sqlJsCDN: 'https://cdnjs.cloudflare.com/ajax/libs/sql.js/1.5.0/'
```

Comment:
I'm not sure if it just plain doesn't work with the latest 1.6.2 version, or if the worker CDN just needs to match the resolved package version.

Reply:
Should we change the loader default until sorted?

Reply:
- sql.js 1.6.2 doesn't seem to have worker files published at `https://cdnjs.cloudflare.com/ajax/libs/sql.js/1.6.2/`
- sql.js 1.6.1 fails to load with error `LinkError: WebAssembly.instantiate(): Import #0 module="a" function="a" error: function import requires a callable`
- sql.js 1.6.0 makes the page hang

sql.js 1.5.0 works. I'll make a PR to pin the GeoPackageLoader to version 1.5.0.